packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js

```diff
@@ -132,4 +175,13 @@ export default class QuanticNumericFacet extends LightningElement {
   preventDefault(evt) {
     evt.preventDefault();
   }
+
+  onApply(evt) {
+    const engine = getHeadlessBindings(this.engineId).engine;
+    engine.dispatch(CoveoHeadless.loadNumericFacetSetActions(engine).deselectAllNumericFacetValues(this.facetId));
+    this.numericFilter.setRange({
+      start: evt.detail.min,
+      end: evt.detail.max
```

Comment:
This is safe if you are sending `undefined` ? For example if I just set a min or set only a max?